config/index.js

```diff
@@ -44,6 +44,9 @@ const config = {
       apiDate: process.env.API_DATE || 'X-API-Date',
       packageVersion: process.env.PACKAGE_VERSION || 'X-Package-Version',
       nodeVersion: process.env.NODE_VERSION || 'X-Node-Version'
+    },
+    geek_joke_api: {
+      url: 'https://geek-jokes.sameerkumar.website/api'
```

Comment:
here we can use an environment variable, what do you think ?

Reply:
Agree, I had been forget it.